_ingest-pipelines/processors/json.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: default
+title: JSON
+parent: Ingest processors
+nav_order: 170
+---
+
+# JSON processor
+
+The `json` processor serializes a string-valued field into a map of maps, which can be useful for various data processing and enrichment tasks.
```

Comment:
Is "string-valued" the right phrasing here, or should it just be "string value field"?